includes/class-sensei-course-progress.php

```diff
@@ -74,36 +58,55 @@ class Sensei_Course_Progress {
 	 * @since   1.0.0
 	 * @return  void
 	 */
-	public function __construct ( $file, $version = '1.0.0' ) {
-		$this->_version = $version;
-		$this->_token = 'sensei_course_progress';
+	public function __construct() {
+		$this->_version = SENSEI_COURSE_PROGRESS_VERSION;
+		$this->_token   = 'sensei_course_progress';
 
-		$this->file = $file;
-		$this->dir = dirname( $this->file );
-		$this->assets_dir = trailingslashit( $this->dir ) . 'assets';
-		$this->assets_url = esc_url( trailingslashit( plugins_url( '/assets/', $this->file ) ) );
+		$this->assets_dir = trailingslashit( dirname( SENSEI_COURSE_PROGRESS_PLUGIN_FILE ) ) . 'assets';
+		$this->assets_url = esc_url( trailingslashit( plugins_url( '/assets/', SENSEI_COURSE_PROGRESS_PLUGIN_FILE ) ) );
 
 		$this->script_suffix = defined( 'SCRIPT_DEBUG' ) && SCRIPT_DEBUG ? '' : '.min';
 
-		register_activation_hook( $this->file, array( $this, 'install' ) );
+		register_activation_hook( SENSEI_COURSE_PROGRESS_PLUGIN_FILE, array( $this, 'install' ) );
 
-		// Load frontend CSS
-		add_action( 'wp_enqueue_scripts', array( $this, 'enqueue_styles' ), 10 );
+		// Handle localisation.
+		$this->load_plugin_textdomain();
```

Comment:
Should this move to `init` as well?

Reply:
Yup, hadn't pushed it (I'm doing this to all PRs)